sbi/inference/snpe/snpe_a.py

```diff
@@ -49,21 +58,16 @@ def __init__(
                 thus be used for shape inference and potentially for z-scoring. It
                 needs to return a PyTorch `nn.Module` implementing the density
                 estimator. The density estimator needs to provide the methods
-                `.log_prob` and `.sample()`.
-                Note that until the last round only a single (multivariate) Gaussian
-                component is used for training (see Algorithm 1 in [1]). In the last
-                round, this component is replicated `num_components` times, its parameters
-                are perturbed with a very small noise, and then the last training round
-                is done with the expanded Gaussian mixture as estimator for the
-                proposal posterior.
+                `.log_prob` and `.sample()`. Note that until the last round only a
+                single (multivariate) Gaussian component is used for training (see
+                Algorithm 1 in [1]). In the last round, this component is replicated
+                `num_components` times, its parameters are perturbed with a very small
+                noise, and then the last training round is done with the expanded
+                Gaussian mixture as estimator for the proposal posterior.
             num_components:
                 Number of components of the mixture of Gaussians. This number is set to
                 1 before running Algorithm 1, and then later set to the specified value
                 before running Algorithm 2.
-            num_rounds: Total number of training rounds. For all but the last round, Algorithm 1
-                from [1] is executed. For last round, Algorithm 2 from [1] is executed once.
-                By default, `num_rounds` is set to 1, i.e. only Algorithm 2 is executed once
-                without training the proposal prior using Algorithm 1.
```

Comment:
Isn't the old doc string correct? The other doc strings in this file have the "old" style for multiline stings.

Reply:
hmm, I'm not sure. Maybe my vscode settings are messed up, no worries.